### PR TITLE
🔶 Update Postgresql to 10.6 on Travis

### PR DESCRIPTION
**Pull request by dodobas**

### CODE REVIEW

File: indexer.go [hunk 1]
```diff
@@ -357,34 +357,34 @@ SELECT org_id, id, modified_on, is_active, row_to_json(t) FROM (
...
-                                 else '{}' :: jsonb
-                                 end ||
-                                 jsonb_build_object('field', key) || value as value
-                          from jsonb_each(contacts_contact.fields)
-                        ) state_value
-                 ) as district_value
+                       select case
+                    when value ? 'ward'
+                      then jsonb_build_object(
+                        'ward_keyword', trim(substring(value ->> 'ward' from  '(?!.* > )([\w ]+)'))
```
Comment (rowanseymour) [inline indexer.go:363]: what's the need for these changes?

Comment (dodobas) [reply indexer.go:363]: Postgresql 10 changed the way how set functions are evaluated, just search for `Change the implementation of set-returning functions appearing in a query's SELECT list` in https://www.postgresql.org/docs/10/release-10.html

so replaced set returning function of `regexp_matches` with a value returning function `substring` (with a modified regex)

**Comment by nicpottier** [comment]
Does this ONLY work on 10? Or are these changes backwards compatible to 9.6. We really want something that work on both or deployment and management is going to be painful.

**Comment by dodobas** [comment]
no, this also WORKS on 9.6

**Pull request merged**